### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

### DISCUSSION THREAD

**Discussion by sachin-rock** [community/community#16795] [Copilot Conversations] [answered]
GitHub Copilot could not connect to server. Extension activation failed: "User not authorized" i am getting this error

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
